### ISSUE TITLE
Ashwalker den not being created often enough

### ISSUE THREAD

**Issue by thegrb93** [ParadiseSS13/Paradise#23718]
### Exploit Reports

- [X] I confirm this issue is not an exploit. (Required)

### BYOND Version

515.1623

### Issue Description

Went 5 rounds without an ashwalker den. Really sad for people who like to play them.

### What did you expect to happen?

Maybe 50% to 60% spawn rate, so every other round.

### What happened instead?

Went 5 rounds without an ashwalker den.

### Why is this bad/What are the consequences?

Lavaland is boring for miners and for ashwalker enjoyers

### Steps to reproduce the issue.

N/A

### When did the problem start happening?

Not sure

**Comment by thegrb93** [comment]
Might be caused by https://github.com/ParadiseSS13/Paradise/commit/6e0905f6cfb2cf1ea7f46d1e5b9a2beb899fdf07

**Comment by PIerreLouisH** [comment]
To me it look like less of an issue and more  of an "intended tm":
As you quoted it yourself, the ruin spawning system got a "budget", the higher the cost the less likely it is to have an ash walker nest.
And 20 is indeed quite expensive.

```
/datum/map_template/ruin/lavaland/ash_walker
	name = "Ash Walker Nest"
	id = "ash-walker"
	description = "A race of unbreathing lizards live here, that run faster than a human can, worship a broken dead city, and are capable of reproducing by something involving tentacles? \
	Probably best to stay clear."
	suffix = "lavaland_surface_ash_walker1.dmm"
	allow_duplicates = FALSE
	cost = 20 // Not auto due to the nature of walkers
```

We are quite able to make into `always_place = TRUE`  (unliekly due to "nature of walkers") or simply lower the cost so the rate of spawn is higher, but any of those would require balance team (or equivalent) approval.

**Comment by thegrb93** [comment]
There is also https://github.com/ParadiseSS13/Paradise/blob/79e7a7b07528f23d9791213587e0f9a0afda4c75/code/datums/ruins.dm#L12 which can increase its likelyhood of getting picked sooner.

**Comment by thegrb93** [comment]
I made a simulation and tried out the probabilities.

It looks like the additional ghost role spawners shouldn't cause any differences as I was thinking. Ashwalkers just have always had somewhere around a 28% chance of spawning.

Changing placement_weight to 3 increases that to 62% chance.

**Comment by DGamerL** [comment]
Yeah this isn't really an issue, this is intended because of Ashwalkers being essentially a free antag roll, meaning that in the PR to make certain ghost spawners always spawn, Ashwalkers were left out due to a design team request.
If you have any other questions you can always ask the design team about it!

**Comment by thegrb93** [comment]
@DGamerL they specifically requested that it doesn't _always_ spawn. Nothing says increasing probability is off the table but whatever.

**Comment by DGamerL** [comment]
> @DGamerL they specifically requested that it doesn't _always_ spawn. Nothing says increasing probability is off the table but whatever.

Yup, that's not the same
But an issue report is not the place to discuss that, feel free to talk with design team members about increasing the chance for ashwalkers to spawn, there is a list of them in the discord, if you're in there
